case-lib/lib.sh

```diff
@@ -768,4 +768,14 @@ set_alsa_settings()
     esac
 }
 
+reset_PGA_volume()
+{
+    # set all PGA* volume to 0dB
+    amixer -Dhw:0 scontrols | sed -e "s/^.*'\(.*\)'.*/\1/" |grep PGA | \
```

Comment:
You don't need the `\` backslash, ending a line with `|  ` is enough

Reply:
```suggestion
    amixer -Dhw:0 scontrols | sed -e "s/^.* '\(.*\)',.*/\1/" |grep PGA | 
```

Reply:
Updated.